src/index.html

```diff
@@ -11,7 +11,97 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset class="box">
+        <legend>Personal informatio</legend>
+        <div class="form-field" >
+          <label>
+            Surname:
+            <input type="text">
+          </label>
+        </div>
+        <div class="form-field">
+          <label>Name:
+            <input type="text">
+          </label>
+        </div>
+        <div class="form-field">
+          <label>How old are you?
+            <input type="text"
+            min="1"max="100"placeholder="12">
+          </label>
+        </div>
+        <div class="form-field">
+          <label>Full date of birth: 
+            <input type="date"
+            placeholder="mm / dd / yyyy">
```

Comment:
fix formatting of inputs everywhere

```suggestion
            <input
              type="date"
              placeholder="mm / dd / yyyy"
            >
```